src/applications/sign-in-changes/components/AccountSwitch.jsx

```diff
@@ -24,12 +24,13 @@ export default function AccountSwitch({ userEmails }) {
   const userHasLogingov = userEmails.logingov;
   const userHasBoth = userHasIdme && userHasLogingov;
   const headingText = userHasLogingov ? 'Login.gov' : 'ID.me';
+  const strongTag = (
+    <strong>{userHasBoth ? 'Login.gov or ID.me' : headingText}</strong>
+  );
   return (
     <div>
       <h2 className="vads-u-margin-y--0" id="accountSwitchH2">
-        Start using your{' '}
-        <strong>{userHasBoth ? 'Login.gov or ID.me' : headingText}</strong>{' '}
-        account now
+        Start using your {strongTag} account now
```

Comment:
Sorry to do this to you 😓 

When I pulled this down it the Mac VoiceOver was still saying this:
`Start using your, level 1, CSP, level 1, account now`.

I had to make the whole h2 text an interpolated string to make it work which is weird 🤔 but the VoiceOver read correctly like this.

```jsx
const cspText = userHasLogingov ? 'Login.gov' : 'ID.me';
const headingText = `Start using your ${userHasBoth ? 'Login.gov or ID.me' : cspText} account now`;
```